gui/scripts/syncthing/core/controllers/syncthingController.js

```diff
@@ -754,12 +753,12 @@ angular.module('syncthing.core')
             $scope.saveDeviceConfig($scope.currentDevice);
         };
 
-        $scope.addNewDeviceID = function (device) {
+        $scope.addNewdeviceId = function (device) {
```

Comment:
This is spurious. But also the only such thing I can see which is impressive! :)